### DISCUSSION TITLE
Slider Step

### DISCUSSION THREAD

**Discussion by swtb3** [pyvista/pyvista#4534] [Q&A] [answered]
Hello,

Is it possible to set the step of the slider widget so that it increases in set amounts. i.e. in steps of 1

**Comment by banesullivan** [comment]
I think this is a missing feature in VTK's `vtkSliderRepresentation2D` / `vtkSliderWidget`

**Reply by swtb3** [reply, marked as answer]
I have found a way to do it.

`def callback(value, slider):
    slider.GetSliderRepresentation().SetValue(int(value))`

Just be sure to enable the pass_widget parameter when adding the slider.

**Reply by banesullivan** [reply]
Fantastic! That's a good trick
